utils/autotailor

```diff
@@ -110,10 +111,10 @@
             change.set("idref", self._full_var_id(varname))
             change.text = str(value)
 
-        if location == "-":
-            location = sys.stdout.buffer
-
-        ET.ElementTree(root).write(location)
+        root_str = ET.tostring(root)
+        pretty_xml = xml.dom.minidom.parseString(root_str).toprettyxml()
+        with open(location, "w") if location != "-" else sys.stdout as f:
```

Comment:
This will close `sys.stdout` and break any future output operation of the script. There are none ATM, and probably will never be. So I think it is OK. Just making a note.

Also, some mocks (that are used in for example bpython instead of real sys.stdout) do not support the context manager protocol. Also just a note.